component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActionsLegacy.java

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
This is the original actions class  AuditEntryActions.  There were no changes other than renaming the class.